my_enumerable.rb

```diff
@@ -0,0 +1,15 @@
+module MyEnumerable
+  def all?(&block)
+    each { |elem| return false unless block.call(elem) }
+    true
+  end
+
+  def any?(&block)
+    each { |elem| return true if block.call(elem) }
+    false
+  end
```

Comment:
- [ ] When I ran the test command the app crashed with the error undefined method `each'. Kindly take a look at the screenshot below, the green underline is the test command used and the yellow underline is the error feedback. 👍🏽

<img width="1382" alt="Screenshot 2023-11-02 at 8 16 10 PM" src="https://github.com/Megagig/Ruby-Enumerable/assets/50941074/6b43e66a-e69b-45bc-bb10-5278eaa29f1e">